lib/tapioca/compilers/symbol_table/symbol_generator.rb

```diff
@@ -221,7 +221,7 @@ def compile_props(constant)
 
         sig { params(constant: Module).returns(String) }
         def compile_enums(constant)
-          return "" unless constant < T::Enum
+          return "" unless T::Enum > constant
```

Comment:
It seems we're just moving the problem around, can't we call directly the definition from `Object`?

Reply:
Yes, there is still some risk but much much less so since `T::Enum` is not a random type, but `constant` is. I'd rather not use the base method if we don't have to since it is much slower to do it.